src/main.rs

```diff
@@ -89,3 +89,19 @@ fn rocket() -> _ {
         }))
         .attach(CORS)
 }
+// TODO: Check the database is up and running before starting running these tests
```

Comment:
Maybe we can extract the check performed in the db init process to a fn that can be called externally?